lib/cocoapods/installer/user_project_integrator/target_integrator/xcconfig_integrator.rb

```diff
@@ -97,7 +95,7 @@ def self.print_override_warning(pod_bundle, target, config, key)
             ]
             message = "The `#{target.name} [#{config.name}]` " \
               "target overrides the `#{key}` build setting defined in " \
-              "`#{pod_bundle.xcconfig_relative_path(config.name)}'. " \
+              "`#{pod_bundle.relative_pods_root_path + pod_bundle.xcconfig_relative_path(config.name)}'. " \
```

Comment:
could this could be pulled out into a function? since its repeated multiple times